project-base/src/Shopsys/ShopBundle/Migrations/Version20190918155540.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+/**
+ * This migration changes the default currencies settings - it should be ran on the clean project-base installation only!
+ */
+class Version20190918155540 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('UPDATE currencies SET exchange_rate = 0.04 WHERE code = \'CZK\'');
```

Comment:
I think, that exchange rate for CZK should stay 1 and exchange rate for EUR should be 25